main.tf

```diff
@@ -274,6 +274,13 @@ resource "google_compute_backend_service" "default" {
           include_named_cookies  = each.value.cdn_policy.cache_key_policy.include_named_cookies
         }
       }
+
+      dynamic "bypass_cache_on_request_headers" {
+        for_each = toset(each.value.cdn_policy.bypass_cache_on_request_headers) != null ? [1] : []
```

Comment:
@hhollenstain Is this logic correct? Assuming you want to create multiple blocks of `bypass_cache_on_request_headers` depending on how many values r passed in your list. I think logic should be as follows:

```
for_each = toset(each.value.cdn_policy.bypass_cache_on_request_headers) != null ? each.value.cdn_policy.bypass_cache_on_request_headers : []
```

Reply:
Yep, you are correct. I will fix it up.